decidim-conferences/spec/system/conference_speakers_spec.rb

```diff
@@ -60,4 +60,41 @@
       end
     end
   end
+
+  context "when admin publishes and unpublishes a speaker" do
```

Comment:
The logic is a bit weird in this specs. Let's simplify it 🙏🏽 

This is the structure that I have in mind: 

```ruby
describe "publication of conference speakers" do
  context "when there are some unpublished speakers" do
    it "is not shown in the public list" do
      skip
    end

    it "publishes the speaker" do
      skip
    end
  end

  context "when there are some published speakers" do
    it "is shown in the public list" do
      skip
    end

    it "unpublishes the speaker" do
      skip
    end
  end
end
```

Can you implement them 🙏🏽 ?

Reply:
Done